articles/security/index.md

```diff
@@ -17,7 +17,8 @@ description: Learn about various topics regarding how to handle sensitive data w
 
 <%= include('../_includes/_topic-links', { links: [
   'security/store-tokens',
-  'tokens'
+  'tokens',
+  'security/blacklisting-attributes'
```

Comment:
It's good that you added this article in the index, but maybe it should be on its own and not grouped under Tokens.

Reply:
Not fixed